packages/paste-core/components/typography/src/Heading.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Text} from '@twilio-paste/text';
+
+export type asTags = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'div' | 'label' | 'span';
+export type HeadingStyle =
+  | 'headingStyle10'
+  | 'headingStyle20'
+  | 'headingStyle30'
+  | 'headingStyle40'
+  | 'headingStyle50'
+  | 'headingStyle60';
+
+export interface HeadingProps {
```

Comment:
I'm a doof, remove export and renaming this to `Heading`

Then when you do `export {Heading}` at the bottom it will export both this and the const.

Reply:
Reverted that change.